Documentation/ceph-pool-crd.md

```diff
@@ -179,6 +182,9 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
 * `mirroring`: Sets up mirroring of the pool
   * `enabled`: whether mirroring is enabled on that pool (default: false)
   * `mode`: mirroring mode to run, possible values are "pool" or "image" (required). Refer to the [mirroring modes Ceph documentation](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring) for more details.
+  * `snapshotSchedule`: schedule snapshot at the pool level **only**
+    * `interval`: periodicity of the snapshots. The interval can be specified in days, hours, or minutes using d, h, m suffix respectively.
+    * `startTime`: optional, determines at what time the snapshot process starts. It can be specified using the ISO 8601 time format.
```

Comment:
> * How does the action of taking a snapshot impact the cluster? Does it pause the IO or just a brief perf impact? A brief note on this may help the user understand why they should care about the startTime.

Taking a snapshot is not disruptive so it's really up to them hence `startTime` being optional.

> * What is the default?

Probably the time when it was setup. @dillaman, what's the default start time for snapshot scheduling if nothing is set? Thanks!

Reply:
> Probably the time when it was setup. @dillaman, what's the default start time for snapshot scheduling if nothing is set? Thanks!

00:00:00 UTC